docs/custom_schemas/h5md/explanation-H5MD-NOMAD.md

```diff
@@ -0,0 +1,1342 @@
+
+* `n_smooth`
+:   number of bins over which the running average was computed for `value`.
+
+* `type`
+:   Allowed values of `molecular` or `atomic`. Categorizes if the observable is calculated at the molecular or atomic level.
+<!-- TODO - not sure if this is useful -->
+
+* `error_type`
+:   describes the type of error reported for this observable. Examples: `Pearson correlation coefficient`, `mean squared error`.
+
+* `errors`
+:   value of the error at each bin. Can be multidimensional with corresponding label stored in `error_labels`.
+
+* `error_labels`
+:   describes the error along individual dimensions for multi-D errors.
+
+* `<custom_dataset>`
+:   additional metadata may be given as necessary.
+<!-- TODO - Is this really parsed?! -->
+
+<a id="time_correlation_observable_anchor"></a>
+
+`time_correlation`
+:   An observable that is computed by calculating correlations between configurations in time, with the following general structure:
+
+    observables
+     \-- <time_correlation_subgroup>
+     |    \-- <label_1>
+     |    |    +-- type: "time_correlation"
+     |    |    \-- (direction): String[]
+     |    |    \-- (n_times): Integer[]
+     |    |    \-- times: Float[n_times][]
+     |    |    \-- value: <type>[n_bins][]
+     |    |    \-- (type): String[]
+     |    |    \-- (error_type): String[]
+     |    |    \-- (errors): Float[n_bins]
+     |    |    \-- (error_labels): String[]
+     |    |    \-- (<custom_dataset>): <type>[]
+     |    \-- ...
+     \-- ...
+
+* `label`
+:   describes the particles involved in determining the property. For example, for a radial distribution function between particles of type `A` and `B`, `label` might be set to `A-B`
+
+* `direction`
+:   allowed values of `x`, `y`, `z`, `xy`, `yz`, `xz`, `xyz`. The direction/s used for calculating the correlation function.
+
+* `n_times`
+:   number of times windows for the calculation of the correlation function. Can also be inferred from dimensions of `times`.
+
+* `times`
+:   time values used for calculating the correlation function (i.e., &Delta;t values).
+
+* `value`
+:   value of the calculated correlation function at each time.
+
+* `type`
+:   Allowed values of `molecular` or `atomic`. Categorizes if the observable is calculated at the molecular or atomic level.
+<!-- TODO - not sure if this is useful -->
+
+* `error_type`
+:   describes the type of error reported for this observable. Examples: `Pearson correlation coefficient`, `mean squared error`.
+
+* `errors`
+:   value of the error at each bin. Can be multidimensional with corresponding label stored in `error_labels`.
+
+* `error_labels`
+:   describes the error along individual dimensions for multi-D errors.
+
+* `<custom_dataset>`
+:   additional metadata may be given as necessary.
+<!-- TODO - Is this really parsed?! -->
+
+A list of standardized observables can be found [HERE](reference-H5MD-NOMAD.md#standardized-observables-in-h5md-nomad).
```

Comment:
Maybe instead of "HERE", use directly the name of the section, something like "Reference - H5MD-NOMAD > Standardized observables in H5MD-NOMAD"

Reply:
done.